wordpress-mobile/WordPress-iOS
# Media Library: Show uploading progress in cells

This PR adds progress updates to cells in the media library when content is being uploaded asynchronously:

![overlays-1](https://user-images.githubusercontent.com/4780/32960177-ef12cef0-cbbb-11e7-8e28-53eceefa1921.gif)

We show indeterminate spinners whilst the media items are being processed locally, then discrete progress during the upload, then indeterminate again once we hit 99% progress and the item is being processed on the remote end.

To test:

* Try uploading items using the async feature in the media library, and check the cells display correctly
* Try scrolling up and down the collectionview during an upload
* Try backing out of the media library and coming back in during an upload
* Check that normal uploads still work as expected
